### DISCUSSION TITLE
Can't send proof of academic status

### DISCUSSION THREAD

**Discussion by MaximilianVollmer** [community/community#54053] [GitHub Education] [answered]
### Hi everyone,

Hello everyone!
So I have some trouble with sending the proof of my academic status I always get this error message:
![image](https://user-images.githubusercontent.com/106180729/235320880-ecb95128-2751-4e2d-b31e-98b2b13b2456.png)
I've went trough my user profile multiple times and it seems complete. I had tried it with my real name as my username and I also wrote a short Bio. But I still get this message.

I also dont have a physical copy of my proof so I cant take a picture with my device as said below in the picture.

So what exactly am I missing to apply for the Education Pack?

**Comment by samus-aran** [comment, marked as answer]
Hi @MaximilianVollmer 

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying. 👈 

Let us know how you get on 👍

**Reply by MaximilianVollmer** [reply]
Thank you @samus-aran ,
Either the english translation of my proof or the relogin did the trick. 👍

**Comment by salernogiuseppe** [comment]
HI,
I applied to join the gitHub academy as a university student, but I had not activated my billing information before registering.
How do I resume registration?
